experimental/tzif/src/lib.rs

```diff
@@ -2,7 +2,10 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-//! A parser for TZIF files inteded for use with the ICU4X DataProvider.
+//! A parser for [Time Zone Information Format (`TZif`)](https://tools.ietf.org/id/draft-murchison-tzdist-tzif-00.html) files.
+//!
+//! Resources to generate `TZif` files are provided by the [IANA database](https://www.iana.org/time-zones).
+//! `TZif` files are also included in some operating systems.
```

Comment:
Request: Add a crate-level example so that when you open the docs page, you see some examples of how to use this crate. Right now the crate-level page does not have much info.

http://icu4x-pr-artifacts.storage.googleapis.com/gha/838104e4e728b33aa8889331ec3b161fe4fd1139/docs/tzif/index.html

This can be done in a follow-up if you prefer.

Reply:
Good idea, thanks. I will do this in the PR that moves everything to `utils`.